src/assets/scripts/client/InputController.js

```diff
@@ -715,6 +715,16 @@ export default class InputController {
 
                 return true;
             }
+            case PARSED_COMMAND_NAME.AIRAC: {
+                const airacCycle = AirportController.getAiracCycle();
+                if (airacCycle === null) {
```

Comment:
add empty line before if

Reply:
done

Reply:
I've been told to use `if (!airacCycle)` in situations like this, in case we somehow end up with other falsey values like undefined.

Reply:
done